app/domain/types.ts

```diff
@@ -21,7 +21,8 @@ export type Symptoms = {
 
 export enum Sex {
   MALE = 'MALE',
-  FEMALE = 'FEMALE'
+  FEMALE = 'FEMALE',
+  X = 'X'
```

Comment:
Yeah it's probably time we add a third option here. Could it maybe be `OTHER` instead? We also need to add the text to all the existing locales. Not sure which text we should go for, "Other", "I prefer not to say", or something else

Reply:
I used "Other" instead of "X" to be more gender inclusive.